Software/Development/SemesterProject/App/SVModulator.c

```diff
@@ -48,7 +48,7 @@ void runSVM(alphaBetaObject voltageRef)
 {
     float Vdc, VdcInverse;
     int sector;
-    handleMaxDuty();
+    MAX_DUTY = getMaxDuty();
```

Comment:
It's misleading to have this variable with capital letters

Reply:
agree, fixed